lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation.rb

```diff
@@ -40,7 +47,14 @@ def show_also_part_of_step_nav?
       end
 
       def related_links
-        step_by_step_navs = active_step_by_step? ? [active_step_by_step] : step_navs
+        step_by_step_navs = if active_step_by_step?
+                              [active_step_by_step]
+                            elsif secondary_step_by_step?
+                              step_navs_combined_list
```

Comment:
`step_navs_combined_list` also contains `related_to_step_navs`.  Are `related_to_step_navs` supposed to show up in `related_links`? What happens if a content item is "related_to" one step nav and "secondary content" on another?

Reply:
In the current solution, both will show up. In my opinion, this is the correct behaviour. The user should be able to see the secondary step navs linked as well as any related step navs.

Reply:
Is this something we should double check with Sam?

Reply:
Yes, I will raise with Sam.